packages/text-input-react/src/TextArea.tsx

```diff
@@ -13,101 +13,111 @@ interface Props extends BaseProps {
     autoExpand?: boolean;
 }
 
-export const TextArea = ({
-    id,
-    variant,
-    inverted,
-    label,
-    className,
-    helpLabel,
-    errorLabel,
-    rows = 7,
-    placeholder = " ",
-    autoExpand = false,
-    forceCompact,
-    ...restProps
-}: Props) => {
-    const componentClassName = classNames("jkl-text-input jkl-text-area", className, {
-        "jkl-text-input--compact": forceCompact,
-        "jkl-text-input--inverted": inverted,
-    });
-    const [uid] = useState(id || `jkl-text-area-${nanoid(8)}`);
-    const [supportId] = useState(`jkl-support-label-${nanoid(8)}`);
+export const TextArea = forwardRef<HTMLTextAreaElement, Props>(
+    (
+        {
+            id,
+            variant,
+            inverted,
+            label,
+            className,
+            helpLabel,
+            errorLabel,
+            rows = 7,
+            placeholder = " ",
+            autoExpand = false,
+            forceCompact,
+            ...restProps
+        },
+        ref,
+    ) => {
+        const componentClassName = classNames("jkl-text-input jkl-text-area", className, {
+            "jkl-text-input--compact": forceCompact,
+            "jkl-text-input--inverted": inverted,
+        });
+        const [uid] = useState(id || `jkl-text-area-${nanoid(8)}`);
+        const [supportId] = useState(`jkl-support-label-${nanoid(8)}`);
 
-    const [textAreaFocused, setTextAreaFocused] = useState(false);
-    const textAreaRef = useRef<HTMLTextAreaElement>(null);
-    const textAreaContentRows = (restProps.value || "").split("\n").length;
+        const [textAreaFocused, setTextAreaFocused] = useState(false);
+        const textAreaRef = (ref as RefObject<HTMLTextAreaElement>) || useRef<HTMLTextAreaElement>(null);
```

Comment:
Kan du gjøre dette? Tror hooks må være på topnivå og i samme rekkefølge for alle renders uansett, her er det ikke det.

```suggestion
        const internalRef  = useRef<HTMLTextAreaElement>(null);
        const textAreaRef = (ref as RefObject<HTMLTextAreaElement>) || internalRef;
```